src/App.css

```diff
@@ -2,7 +2,7 @@
   max-width: 1280px;
   margin: 0 auto;
   padding: 2rem;
-  text-align: center;
+  /* text-align: center; */
```

Comment:
Not 100% sure if necessary, will check when working on the table itself.
Probably should not include but this will remind me later.